Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -163,6 +163,13 @@ extension BrowserViewController: WKNavigationDelegate {
         info["title"] = webView.title
 
         notificationCenter.postNotificationName("LocationChange", object: self, userInfo: info)
+
+        UIAccessibilityPostNotification(UIAccessibilityScreenChangedNotification, nil)
```

Comment:
According to <a href="https://developer.apple.com/library/ios/documentation/UIKit/Reference/UIAccessibility_Protocol/index.html#//apple_ref/c/data/UIAccessibilityScreenChangedNotification">the docs</a>, UIAccessibilityScreenChangedNotification expects a parameter, "which is either an NSString object that VoiceOver speaks or an accessibility element that VoiceOver moves to".

Could passing nil here be causing the strange behavior you're seeing? What happens if you explicitly choose the element?